### PR TITLE
Enable SCSS whitespace rules

### PR DESCRIPTION
**Pull request by joshlory**
Fix & enable [SpaceAfterPropertyColon](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceafterpropertycolon) and [SingleLinePerProperty](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#singlelineperproperty) `scss-lint` cops.

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -251,10 +252,10 @@ a.header_status_bar:hover {
...
 .main {
   padding: 10px;
-  a:link {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color;}
-  a:visited {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color;}
-  a:hover {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color; background-color: transparent;   text-decoration: underline; }  /* mouse over link */
-  a:active {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color; background-color: transparent;}  /* selected link */
+  a:link {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color;}
+  a:visited {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color;}
+  a:hover {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color; background-color: transparent;   text-decoration: underline; }  /* mouse over link */
+  a:active {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color; background-color: transparent;}  /* selected link */
```
Comment (joshlory) [inline dashboard/app/assets/stylesheets/application.css.scss]: There's a config option `allow_single_line_rule_sets` we can set to `false` if you think these should also be on separate lines.

Comment (breville) [inline dashboard/app/assets/stylesheets/application.css.scss]: It seems reasonable to break these out to separate lines too.  Like, they're special, but not _that_ special :)

File: dashboard/app/assets/stylesheets/level.css.scss [hunk 2]
```diff
@@ -24,7 +24,11 @@ li.level_name {
   width: 100%;
   height: 100%;
   position: absolute;
-  top: 0; bottom: 0; right: 0; left: 0; /* fill parent */
+  // Fill parent.
```
Comment (breville) [inline dashboard/app/assets/stylesheets/level.css.scss:27]: This makes me realise how little we generally comment on our CSS.

**Comment by breville** [comment]
lgtm

File: dashboard/app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (joshlory) [inline dashboard/app/assets/stylesheets/application.css.scss]: Fixed in 5c9011027374e976a5556dadf8f9e173ba9ef066.

**Comment by joshlory** [comment]
FYI @ashercodeorg.

**Comment by ashercodeorg** [comment]
LGTM. Thanks!

**Pull request merged**